### ISSUE TITLE
Force tablet mode crash

### ISSUE THREAD

**Issue by BugReportI** [uazo/bromite-buildtools#227]
Enabling the force tablet mode in Chromium Version 114.0.5735.91 causes the browser to keeps crashing.

If anyone else wants to test this then backup your app data first or you will lose all your data, since the app keeps crashing after enabling this option, so you will not be able to turn the setting off without uninstalling and I tried downgrading the app it says 'App not installed as package appears to be invalid'.

**Comment by uazo** [comment]
thank you, I have known for a while.
my idea is to drop that patch, which hasn't worked for years.

**Comment by BugReportI** [comment]
Okay, thanks for the update on what going to happen with this feature.

Kind of sad, since I like that it made the UI more accessible for me.

**Comment by uazo** [comment]
> UI more accessible for me

let me get this straight.
That patch never worked for me, how did the ui look to you?

**Comment by BugReportI** [comment]
The patch works in version 113 and 112.

When open the tabs they look huge without tablet mode and in landscape mode it is easier bookmark current tab since I do not have to open the menu button. The bookmark button is shown in the search bar.

I will try to get the screenshots to better explain, bit later.

I do not want to do it now since I just found the solution to upgrading the app with my data by trying all combination of settings until it did not crash, since I did not want lost my app data.

**Comment by uazo** [comment]
> I will try to get the screenshots to better explain, bit later.

OK, I'll wait

> since I did not want lost my app data.

tell me what device you have that I'll try on browserstack if it's there.

**Comment by BugReportI** [comment]
It is Sony Xperia 1 III with official latest LineageOS 20.0  and has Lygisk (Magisk fork).

**Comment by uazo** [comment]
no is not present, too bad.
can you help me look for a 6.5" device with a resolution of 1644 x 3840 pixels and 643 ppi (643 really? is a tablet or phone?): https://www.browserstack.com/list-of-browsers-and-platforms/app_live

**Comment by uazo** [comment]
tested with an oneplus 9, you mean these?

![image](https://github.com/uazo/bromite-buildtools/assets/29201891/5d680083-2117-4cb7-9610-54538df8564e)

nteresting, but it has nothing to do with the patch! that should activate the tabs as in the tablet, that's why I said that the patch hasn't worked for years.

but I like those buttons, in fact I've been looking for that very piece of code for a while..
thanks, I think I'll do something with it.

**Comment by BugReportI** [comment]
Well it is a phone since it is this device https://m.gsmarena.com/sony_xperia_1_iii-10712.php.

This device mostly runs in 1080p unless content on screen is 4K compatible, then it runs in 4K.

I think any device that has 6.5 inches display in 1080p should be fine to use, since only Sony makes 4K display phones.

**Comment by BugReportI** [comment]
I 

> tested with an oneplus 9, you mean these?
> 
> ![image](https://user-images.githubusercontent.com/29201891/242854458-5d680083-2117-4cb7-9610-54538df8564e.png)
> 
> nteresting, but it has nothing to do with the patch! that should activate the tabs as in the tablet, that's why I said that the patch hasn't worked for years.
> 
> but I like those buttons, in fact I've been looking for that very piece of code for a while.. thanks, I think I'll do something with it.

I think there were some other things that get enabled with force table mode. I will try to see if I find any others later since it can improve accessibility of the app.

**Comment by BugReportI** [comment]
> tested with an oneplus 9, you mean these?
> 
> ![image](https://user-images.githubusercontent.com/29201891/242854458-5d680083-2117-4cb7-9610-54538df8564e.png)
> 
> nteresting, but it has nothing to do with the patch! that should activate the tabs as in the tablet, that's why I said that the patch hasn't worked for years.
> 
> but I like those buttons, in fact I've been looking for that very piece of code for a while.. thanks, I think I'll do something with it.

Yeah I mean those they are very useful, so I do not have to open menu button every to bookmark a website. I do not know if it is possible to implement them in portrait mode because of how narrow phones how become nowadays.

**Comment by BugReportI** [comment]
Here are how the tabs look without force tab mode in Chromium

Landscape
![Screenshot_20230605-182947_Trebuchet](https://github.com/uazo/bromite-buildtools/assets/120331504/6888e36c-b7f3-4f01-9f72-42b115512d5b)

Landscape with force tab mode
![Screenshot_20230605-184740_Trebuchet](https://github.com/uazo/bromite-buildtools/assets/120331504/1c11758d-2bba-4184-a636-9b46233d2ba5)

In portrait the amount tabs shown is the same with or without force tab mode.

If the tabs could made to be smaller in portrait as well, since they seem bit too big.
![Screenshot_20230605-183001_Trebuchet_1_Edited](https://github.com/uazo/bromite-buildtools/assets/120331504/9bf111d0-15ea-426b-9911-b266abfa6254)
